SCIMAG3D/iocBoot/iocSCIMAG3D-IOC-01/config.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" ?>
+<ioc_config xmlns:xi="http://www.w3.org/2001/XInclude">
+ <config_part>
+  <ioc_desc>3D Magnet</ioc_desc>
+  <ioc_details>3D Vector Magnet System from Scientific Magnetics</ioc_details>
+  <macros>
+    <macro name="LVDCOM_OPTIONS" pattern="^\d?\d?$" description="Options that define how the VI is started. Add selected options together to make the final value. 1 - warn if idle, 2 - start if idle, 4 - stop VIs if started on exit, 6 - stop VI on exit. (Default 1)" />
+	<macro name="LVDCOM_HOST" pattern="^.*$" description="Host on which the vi is running, blank for localhost (Default '')" />
```

Comment:
You make a valid point about their visibility to users if included here, but leaving them in st.cmd allows use in globals.txt   Maybe an XML comment here to this effect? At ISIS we will not generally need to specify username/password on an instrument though